control/control.yaml

```diff
@@ -25,6 +25,8 @@ components:
           $ref: '#/components/schemas/Control.PortCaptureState'
         flow_transmit_state:
           $ref: '#/components/schemas/Control.FlowTransmitState'
+        bgp_contol_state:
```

Comment:
bgpv4_control_state

Reply:
remove the bgpcontrolstate for now - the /results/bgpv4 has all of that information that can easily be aggregated using pandas

Reply:
I have added bgpv4_control_state to control BGP related operations. These are probable step to require this options:

Strat all protocols and traffic. (I think existing FlowTransmitState(state='start') should handle that part)
Stop BGP (I have plan to trigger that through State(BgpControlState(state='stop')))
Flap BGP link (trigger it using State(BgpControlState(state='link_flap')) )
I will remove this section if this is not an actual customer use case.